e2e/static-pod-nad.yml

```diff
@@ -0,0 +1,16 @@
+apiVersion: "k8s.cni.cncf.io/v1"
+kind: NetworkAttachmentDefinition
+metadata:
+  name: bridge-nad
+  namespace: default
```

Comment:
we may not need 'namespace: default' because of 'default'.

Reply:
I don't think the cluster defaults to the "default" namespace after creation. For example, on CI, after creating the cluster, the namespace is set to kind-kind. I think the net-attach-def then gets created in this namespace. For this reason I think line 5 may be needed.